doc/subsystems/disk/sdhc.rst

```diff
@@ -0,0 +1,42 @@
+SDHC disks
+##########
+
+Zephyr includes support for connecting a SDHC card via the SPI bus.
+When used with ELMFS, this allows reading and writing FAT based
+filesystems.
+
+The system has been tested with cards from Samsung, SanDisk, and 4V
+with sizes from 2 GiB to 32 GiB in single partition mode.
+
+MMC and SDSC (<= 2 GiB) cards are not supported and will be ignored.
+
+The system does not support inserting or removing cards, so the card
+must be present at boot and must not be removed.  This may be fixed in
+future releases.
+
+Note that FAT is not power safe so the filesystem may become corrupted
+if power is lost or if the card is removed.
+
+Enabling
+********
+
+The SDHC card slot is configured using Device Tree and should be added
+as a child of a SPI port.  For example, the fragment:
+
+.. code-block:: c
+
+    &spi1 {
+            status = "ok";
+
+            sdhc0: sdhc@0 {
+                    compatible = "zephyr,mmc-spi-slot";
+                    reg = <0>;
+                    status = "ok";
+                    label = "SDHC0";
+                    spi-max-frequency = <24000000>;
+                    gpios = <&porta 27 0>;
+            };
+    };
+
+adds SDHC card slot on `spi1`, uses `PA27` for chip select, and runs
+the SPI bus at 24 MHz once the SDHC card has been initialised.
```

Comment:
``initialized`` (American spelling)

Are there any tips for using the device after it's been initialized?

Reply:
Added a short usage section.  Unfortunately there's no generic FS documentation or sample ATM :(